src/parse_args.ts

```diff
@@ -59,7 +59,12 @@ program
     '--capability <features...>',
     'Enable capabilities through feature flags'
   )
-  .option('--screenshots', 'take screenshot for each step')
+  .addOption(
```

Comment:
Could use a comment here explaining why we have this declared twice. I take it we need the second option for the `--screenshots` option to work with no setting, yes?

Reply:
It was my mistake, i did not remove the next line, it should work without two options.